ncurses/plan.sh

```diff
@@ -1,8 +1,11 @@
 pkg_name=ncurses
 pkg_origin=core
 pkg_version=6.0
+pkg_description="The ncurses (new curses) library"
+pkg_upstream_url=https://www.gnu.org/software/ncurses/
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('ncurses')
```

Comment:
This is not a valid [SPDX license](https://spdx.org/licenses/) string.

The ncurses licensing is quite complicated. http://invisible-island.net/ncurses/ncurses-license.html

I'm not quite certain what we should do here. Other maintainers with license knowledge, chime in?